api/organization.go

```diff
@@ -6,11 +6,12 @@ import (
 )
 
 type Organization struct {
-	ID      uid.ID `json:"id"`
-	Name    string `json:"name"`
-	Created Time   `json:"created"`
-	Updated Time   `json:"updated"`
-	Domain  string `json:"domain"`
+	ID                  uid.ID   `json:"id"`
+	Name                string   `json:"name"`
+	Created             Time     `json:"created"`
+	Updated             Time     `json:"updated"`
+	Domain              string   `json:"domain"`
+	AllowedLoginDomains []string `json:"allowedLoginDomains" note:"domains which can be used with social login for this organization" example:"['example.com', 'infrahq.com']"`
```

Comment:
```suggestion
	AllowedDomains []string `json:"allowedLoginDomains" note:"domains which can be used to log into this organization" example:"['example.com', 'infrahq.com']"`
```

`AllowedDomains` is a better name here, and will eventually be for any non-idp login so we may want to keep the comment general